app/javascript/react/screens/App/Overview/components/Migrations/MigrationsNotStartedList.js

```diff
@@ -1,78 +1,166 @@
 import OverviewEmptyState from '../OverviewEmptyState/OverviewEmptyState';
+import ScheduleMigrationModal from '../ScheduleMigrationModal/ScheduleMigrationModal';
 
 const MigrationsNotStartedList = ({
   migrateClick,
   notStartedPlans,
   loading,
-  redirectTo
+  redirectTo,
+  showConfirmModalAction,
+  hideConfirmModalAction,
+  toggleScheduleMigrationModal,
+  scheduleMigrationModal,
+  scheduleMigrationPlanId,
+  scheduleMigration
 }) => (
-  <Grid.Col xs={12}>
-    <Spinner loading={!!loading}>
-      {notStartedPlans.length > 0 ? (
-        <ListView className="plans-not-started-list" style={{ marginTop: 0 }}>
-          {notStartedPlans.map(plan => (
-            <ListView.Item
-              className="plans-not-started-list__list-item"
-              onClick={() => {
-                redirectTo(`/migration/plan/${plan.id}`);
-              }}
-              actions={
-                <div>
-                  <Button
-                    onClick={e => {
-                      e.stopPropagation();
-                      migrateClick(plan.href);
-                    }}
-                    disabled={loading === plan.href}
-                  >
-                    {__('Migrate')}
-                  </Button>
-                </div>
-              }
-              leftContent={<div />}
-              heading={plan.name}
-              description={plan.description}
-              additionalInfo={[
-                <ListView.InfoItem key={plan.id}>
-                  <Icon type="pf" name="virtual-machine" />
-                  <strong>{plan.options.config_info.vm_ids.length}</strong>{' '}
-                  {__('VMs')}
-                </ListView.InfoItem>
-              ]}
-              key={plan.id}
-            />
-          ))}
-        </ListView>
-      ) : (
-        <OverviewEmptyState
-          title={__('No Migration Plans Not Started')}
-          iconType="pf"
-          iconName="info"
-          description={
-            <span>
-              {__(
-                'There are no existing migration plans in a Not Started state.'
-              )}
-              <br />{' '}
-              {__(
-                'Make a selection in the dropdown to view plans in other states.'
-              )}
-            </span>
-          }
-        />
-      )}
-    </Spinner>
-  </Grid.Col>
+  <React.Fragment>
+    <Grid.Col xs={12}>
+      <Spinner loading={!!loading}>
+        {notStartedPlans.length > 0 ? (
+          <ListView className="plans-not-started-list" style={{ marginTop: 0 }}>
+            {notStartedPlans.map(plan => {
+              const confirmationWarningText = (
+                <React.Fragment>
+                  <p>
+                    {__('Are you sure you want to unschedule plan ')}
+                    <strong>{plan.name}</strong>
+                    {__(' targted to run on ')}
+                    {moment(plan.schedule_time).format('MMMM Do YYYY, h:mm a')}
+                    {__('?')}
```

Comment:
I am aware the PR is still WIP, but here's the thing that we discussed in the i18n context about complete sentences.

The sentence should ideally look like this -
"Are you sure you want to unschedule plan %s targeted to run on %s?"

Reply:
Check out https://github.com/ManageIQ/miq_v2v_ui_plugin/blob/9aa142a0b212750cb7affb286654e0c98b49c1e9/app/javascript/react/screens/App/Overview/components/DeleteInfrastructureMappingConfirmationModal/DeleteInfrastructureMappingConfirmationModal.js#L58 for styled, `sprintf` implementation.

Reply:
yeah! thanks for the 👁 🙇 ❤️